lokinet/lokinet-base.dockerfile

```diff
@@ -1,24 +1,34 @@
-FROM debian:stable AS lokinet-base
+#use argument instead of lsb-release
+ARG DEBIAN_RELEASE=bullseye 
+
+FROM debian:${DEBIAN_RELEASE}-slim AS lokinet-base
 ENV container docker
+
+ENV RELEASE=${DEBIAN_RELEASE:-bullseye}
+#Add oxen public key 
+ADD --chmod=644 --chown=_apt https://deb.oxen.io/pub.gpg /etc/apt/trusted.gpg.d/lokinet.gpg
+
 # set up packages
-RUN /bin/bash -c 'echo "man-db man-db/auto-update boolean false" | debconf-set-selections'
-RUN /bin/bash -c 'apt-get -o=Dpkg::Use-Pty=0 -q update && apt-get -o=Dpkg::Use-Pty=0 -q dist-upgrade -y && apt-get -o=Dpkg::Use-Pty=0 -q install -y --no-install-recommends ca-certificates curl iptables dnsutils lsb-release systemd systemd-sysv cron conntrack iproute2 python3-pip wget'
-RUN /bin/bash -c 'curl -so /etc/apt/trusted.gpg.d/lokinet.gpg https://deb.oxen.io/pub.gpg'
-RUN /bin/bash -c 'echo "deb https://deb.oxen.io $(lsb_release -sc) main" > /etc/apt/sources.list.d/lokinet.list'
-RUN /bin/bash -c 'apt-get -o=Dpkg::Use-Pty=0 -q update && apt-get -o=Dpkg::Use-Pty=0 -q dist-upgrade -y && apt-get -o=Dpkg::Use-Pty=0 -q install -y --no-install-recommends lokinet'
+# not sure if wget, lsb-release and curl are needed (maybe they can be removed to make a smaller image?)
+RUN DEBIAN_FRONTEND=noninteractive \
+    && echo "deb https://deb.oxen.io ${RELEASE} main" > /etc/apt/sources.list.d/lokinet.list \     
+    && echo "man-db man-db/auto-update boolean false" | debconf-set-selections \
+    && apt update -y \
+    && apt full-upgrade -y \
+    && apt install -y --no-install-recommends ca-certificates iptables dnsutils systemd systemd-sysv cron conntrack iproute2 python3-pip \
+    && apt update -y \    
+    && apt install -y --no-install-recommends lokinet \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
why remove `/var/lib/apt/lists/*` ?

Reply:
> why remove `/var/lib/apt/lists/*` ?

Cleaning up - It's a docker best practice.

Reply:
i would advise against doing this in a debian image, regardless of the norms in docker.

Reply:
No harm in doing this. Check [this](https://askubuntu.com/questions/179955/var-lib-apt-lists-is-huge) out